openapi3filter/issue625_test.go

```diff
@@ -0,0 +1,242 @@
+package openapi3filter_test
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/getkin/kin-openapi/openapi3filter"
+	"github.com/getkin/kin-openapi/routers/gorillamux"
+)
+
+func TestIssue625(t *testing.T) {
+
+	anyItemSpec := `
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Sample API
+paths:
+ /items:
+  get:
+    description: Returns a list of stuff
+    parameters:
+    - description: test object
+      explode: false
+      in: query
+      name: test
+      required: false
+      schema:
+        type: array
+        items: {}   ###
```

Comment:
Please make it obvious the parts of these blobs you reuse and the parts you duplicate. 
Maybe introduce a function that takes the missing body as argument and simply returns header + body + footer.

Reply:
fixed. I using strings.ReplaceAll